src/quick-lint-js/crash-handling.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2020  Matthew "strager" Glazar
+// See end of file for extended copyright information.
+
+#ifndef QUICK_LINT_JS_CRASH_HANDLING_H
+#define QUICK_LINT_JS_CRASH_HANDLING_H
+
+#if QLJS_SUBLIME_TEXT_PLUGIN
+
+// If there is for example a program's segmentation fault, the OS will
+// send a signal to the program's process.
+//
+// The program has predefined signal handlers for each type of signal.
+// These signal handlers are called when your specified signal is received.
+// The program can set your own signal handlers.
+//
+// If one of these handlers is called, and inside it, there is a longjmp
+// (for the program to continue the execution), the next time this
+// signal handler be called, the program's process will be killed
+// (or the program will have undefined behavior), but if instead of
+// longjmp a siglongjmp be used, the next time this signal handler
+// be called, the process will not be killed (at least on linux).
```

Comment:
Must fix: We must never try to recover from a signal. If a SIGSEGV happens, the heap might be corrupted, and future lints (which reuse the heap) would crash again or possibly do worse things.

Let the operating system and Sublime Text deal with crashes. We cannot recover from arbitrary crashes from the inside, and we should not even try.